README.md

```diff
@@ -126,75 +126,14 @@ shell access.
  clone the remote master branch, then checkout a specific commit hash.
-- [release tag](https://github.com/jeff1evesque/machine-learning/blob/master/README.md#release-tag):
+- [release tag](https://github.com/jeff1evesque/machine-learning/blob/master/documentation/configuration/setup_clone.rst#release-tag):
  clone the remote branch, associated with the desired release tag.
 
-### Simple clone
-
-```bash
-cd /[destination-directory]
-sudo git clone https://[account]@github.com/[account]/machine-learning.git
-cd machine-learning
-git remote add upstream https://github.com/[account]/machine-learning.git
-```
-
-**Note:** `[destination-directory]` corresponds to the desired directory path,
- where the project repository resides.  `[account]` corresponds to the git
- username, where the repository is being cloned from.  If the original
- repository was forked, then use your git username, otherwise, use
- `jeff1evesque`.
-
-### Commit hash
-
-```bash
-cd /[destination-directory]
-sudo git clone https://[account]@github.com/[account]/machine-learning.git
-cd machine-learning
-git remote add upstream https://github.com/[account]/machine-learning.git
-# stop vagrant
-vagrant halt
-# ensure diffs don't prevent checkout, then checkout hash
-git checkout -- .
-git checkout [hash]
-```
-
-**Note:** the hashes associated with a release, can be found under the
- corresponding tag value, on the [release](https://github.com/jeff1evesque/machine-learning/releases)
- page.
-
-**Note:** `[destination-directory]` corresponds to the desired directory path,
- where the project repository resides.  `[account]` corresponds to the git
- username, where the repository is being cloned from.  If the original
- repository was forked, then use your git username, otherwise, use
- `jeff1evesque`.
-
-### Release tag
-
-```bash
-cd /[destination-directory]
-# clone release tag: master branch does not exist
-sudo git clone -b [release-tag] --single-branch --depth 1 https://github.com/[account]/machine-learning.git [destination-directory]
-git remote add upstream https://github.com/[account]/machine-learning.git
-# create master branch from remote master
-cd machine-learning
-git checkout -b master
-git pull upstream master
-# return to release tag branch
-git checkout [release-tag]
-```
-
-**Note:** `[release-tag]` corresponds to the [release tag](https://github.com/jeff1evesque/machine-learning/tags)
- value, used to distinguish between releases.
-
-**Note:** `[destination-directory]` corresponds to the desired directory path,
- where the project repository resides.  `[account]` corresponds to the git
- username, where the repository is being cloned from.  If the original
- repository was forked, then use your git username, otherwise, use
- `jeff1evesque`.
+ **Note**: various approaches of cloning can be found [here](https://github.com/jeff1evesque/machine-learning/blob/master/documentation/configuration/setup_clone.rst).
```

Comment:
I would a minor language change, simply from the world of web development, it's generally frowned upon, when using words like *here* to signify something, somewhere else:

>
**Note**: various approaches of cloning can be found [here](https://github.com/jeff1evesque/machine-learning/blob/master/documentation/configuration/setup_clone.rst).

To the following:

>
**Note:** various [approaches](https://github.com/jeff1evesque/machine-learning/blob/master/documentation/configuration/setup_clone.rst) can be used to clone the repository, which would cover the possibility of cloning the repository, at a specific commit hash.